Test/expectations/Expect.dfy.expect

```diff
@@ -1,12 +1,12 @@
 
 Dafny program verifier finished with 1 verified, 0 errors
-Program halted: expectation violation
+Program halted: Expect.dfy(21,2): expectation violation
```

Comment:
super-small nit: I'm not a fan of the double colon in the output. :) I can't think of any precedent for this output from other language runtimes off the top of my head though, so I'm not sure exactly what to suggest.

Reply:
Format adjusted per a subsequent email
[Program halted] location: message